src/common/ArtemisService.ts

```diff
@@ -3,63 +3,64 @@
  * Licensed under the MIT License. See License.txt in the project root for license information.
  */
 
-import fetch, {RequestInit} from "node-fetch";
+import fetch, { RequestInit } from "node-fetch";
 import { COPILOT_UNAVAILABLE, US_GEO } from "./copilot/constants";
 import { ITelemetry } from "../client/telemetry/ITelemetry";
 import { sendTelemetryEvent } from "./copilot/telemetry/copilotTelemetry";
 import { CopilotArtemisFailureEvent, CopilotArtemisSuccessEvent } from "./copilot/telemetry/telemetryConstants";
 
-export async function getIntelligenceEndpoint (orgId: string, telemetry:ITelemetry, sessionID:string) {
-    const { tstUrl, preprodUrl, prodUrl } = convertGuidToUrls(orgId);
-    const endpoints = [tstUrl, preprodUrl, prodUrl];
+declare const IS_DESKTOP: string | undefined;
```

Comment:
Unless I'm missing something, declaring this as `const` without setting a value means that this is just always `undefined`, and thus falsey everywhere you use it?

Reply:
I must say this one is a bit mysterious.
https://medium.com/@bhautikbharadava/environment-variables-webpack-config-using-defineplugin-1a7f38e2236e